Documentation update (#99)
eclipse-velocitas/velocitas-docs: content/en/docs/concepts/development_model/_index.md

### SDKs

> Our SDKs, available for different programming languages, are the foundation for the vehicle abstraction provided by the vehicle model and the abstraction of the underlying middleware and communication protocols.
> Besides that, they provide the base classes and utilities for the Vehicle Apps.

Reviewer: Still too long and hard to understand ;-)
```suggestion
Our SDKs, available for different programming languages, are the foundation for the vehicle abstraction provided by the vehicle model. Furthermore, they offer abstraction from the underlying middleware and communication protocols, and they provide the base classes and utilities for the Vehicle Apps.
```

### SDKs

> To reduce the effort required to implement [Vehicle Apps](#vehicle-apps), Velocitas provides a set of SDKs for different programming languages. SDKs are available for Python and C++, further SDKs for Rust and C are planned.

Reviewer: ```suggestion
Our SDKs, available for different programming languages, are the foundation for the vehicle abstraction provided by the vehicle model and the abstraction of the underlying middleware and communication protocols.
Besides that, they provide the base classes and utilities for the Vehicle Apps.
SDKs are available for Python and C++, currently. Further SDKs for Rust and C are planned.
```

Reviewer: "C": Really?

> Next to a Vehicle App abstraction, the SDKs are [Middleware](#middleware-abstraction)-enabled, provide connectivity to the [Data Broker](#data-broker) and contain the ontology in the form of base classes to create [Vehicle Models](#vehicle-models).

Reviewer: "Next to a Vehicle App abstraction, the SDKs are [Middleware](#middleware-abstraction)-enabled"
I don't understand the first part of the sentence

Reviewer 2: Also hardly understandable for me to be honest.. :D

Reviewer: Made a proposal above. This section could be removed then.

- **Polyglot:**
  Vehicle Apps can be written in any programming language. System-level programming languages like Rust and C/C++ are particularly relevant for limited hardware resources found in vehicles, but higher-level languages like Python and JavaScript are also considered for special use cases.

- **OCI-compliant containers:**
  Vehicle Apps are deployed as OCI-compliant containers. The size of these containers should be minimal to fit on constrained devices.

- **Isolation:**
  Each Vehicle App should execute in its own process and should be self-contained with its interfaces and functionality exposed on its own port.

- **Configurations:**
  Configuration information is separated from the code base of the Vehicle App, so that the same deployment can propagate across environments with their respective configuration applied.

- **Disposability:**
  Favor fast startup and support graceful shutdowns to leave the system in a correct state.

- **Observability:**
  Vehicle Apps provide traces, metrics and logs of every part of the application using _Open Telemetry_.

- **Over-the-air updatability:**

>   Vehicle Apps are released to cloud backends like [Pantaris](https://www.etas.com/en/products/pantaris-over-the-air-services.php) and can be updated in vehicles frequently over the air.

Reviewer: "released" or "uploaded"?
```suggestion
  Vehicle Apps are uploaded to cloud backends like [Pantaris](https://www.etas.com/en/products/pantaris-over-the-air-services.php) and can be updated in vehicles frequently over the air.
```
